docs/modules/ROOT/pages/faqs.adoc

```diff
@@ -93,24 +110,24 @@ https://grafana.com/[Grafana] is a visualization tool that helps you make sense
 
 == Pulsar Connector
 
-=== What are the features provided by Cassandra Data Connector that are not supported in `pulsar-sink` or `kafka-sink`?
+=== What are the features provided by Cassandra Source Connector that are not supported in `pulsar-sink`?
```

Comment:
```suggestion
=== What are the features provided by Cassandra Source Connector that are not supported in pulsar-sink?
```

This formatting doesn't work well in headings.

Reply:
Agreed - this is all about `pulsar-sink`.  I've updated this to reflect that.